tests/math/test_matrix_manipulation.py

```diff
@@ -23,6 +23,17 @@
 import pennylane as qml
 from pennylane import numpy as pnp
 
+# Import interfaces
+autograd = pytest.importorskip("autograd")
+tf = pytest.importorskip("tensorflow")
+torch = pytest.importorskip("torch")
+jax = pytest.importorskip("jax")
+jnp = pytest.importorskip("jax.numpy")
+
+# Define a list of dtypes to test
+dtypes = ["complex64", "complex128"]
```

Comment:
I dont think this is used anywhere?

Reply:
kept this because I did some formatting on some of the other unit tests which were importing  torch within the unit tests themselves, which to my understanding, imports should be at the top